### PR TITLE
Add regions for Albania,Denmark,Greece,Iceland,Portugal and Sweden

### LINKED ISSUE #31040: [Issue] Add regions for Albania,Denmark,Greece,Iceland,Portugal and Sweden
This issue is automatically created based on existing pull request: magento/magento2#30959: Add regions for Albania,Denmark,Greece,Iceland,Portugal and Sweden

---------

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Albania,Denmark,Greece,Iceland,Portugal and Sweden states/regions.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by nikolalardev**
### Related Pull Requests
https://github.com/magento-commerce/inventory/pull/42

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Albania,Denmark,Greece,Iceland,Portugal and Sweden states/regions.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#31040: Add regions for Albania,Denmark,Greece,Iceland,Portugal and Sweden

### CODE REVIEW

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForGreece.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            ['GR', 'GR-F', 'Ionía Nísia'],
+            ['GR', 'GR-D', 'Ípeiros'],
+            ['GR', 'GR-B', 'Kentrikí Makedonía'],
+            ['GR', 'GR-M', 'Kríti'],
+            ['GR', 'GR-L', 'Nótio Aigaío'],
+            ['GR', 'GR-J', 'Pelopónnisos'],
+            ['GR', 'GR-H', 'Stereá Elláda'],
+            ['GR', 'GR-E', 'Thessalía'],
+            ['GR', 'GR-K', 'Vóreio Aigaío'],
+
```
Comment (gwharton) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForGreece.php:81]: Should we also be including GR-69 Ágion Óros, self governed region of Greece

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForPortugal.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            ['PT', 'PT-12', 'Portalegre'],
+            ['PT', 'PT-13', 'Porto'],
+            ['PT', 'PT-14', 'Santarém'],
+            ['PT', 'PT-15', 'Setúbal'],
+            ['PT', 'PT-16', 'Viana do Castelo'],
+            ['PT', 'PT-17', 'Vila Real'],
+            ['PT', 'PT-18', 'Viseu'],
+            ['PT', 'PT-19', 'Região Autónoma dos Açores'],
+            ['PT', 'PT-20', 'Região Autónoma da Madeira']
+        ];
```
Comment (gwharton) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForPortugal.php:87]: According to ISO 3166-2, the two autonomous regions in Portugal have region codes PT-20 and PT-30. Can we make these consistent.

Comment (nikolalardev) [reply app/code/Magento/Directory/Setup/Patch/Data/AddDataForPortugal.php:87]: @gwharton Hello, I fix the this two region codes, good catch. Thank you.

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForGreece.php (hunk 1 above)
Comment (nikolalardev) [reply app/code/Magento/Directory/Setup/Patch/Data/AddDataForGreece.php:81]: @gwharton I wasn't sure that should be added, but now is there.

**Comment by gabrieldagama** [comment]
@magento create issue

**Comment by gabrieldagama** [comment]
@magento run all tests

**Review by gabrieldagama — CHANGES_REQUESTED** [review]
Hi @nikolalardev, thanks for contributing, moving this to the correct state.

**Comment by gabrieldagama** [comment]
@magento run all tests

**Comment by gabrieldagama** [comment]
@magento run all tests

**Comment by gabrieldagama** [comment]
Hi @nikolalardev, please follow the exception being thrown on the tests:
```
10:27:05 [Build Functional CE] In ErrorHandler.php line 61:
10:27:05 [Build Functional CE]                                                                                
10:27:05 [Build Functional CE]   [Exception]                                                                  
10:27:05 [Build Functional CE]   Warning: call_user_func() expects parameter 1 to be a valid callback, class  
10:27:05 [Build Functional CE]    'Magento\Directory\Setup\Patch\Data\AddDataForGreece' not found in /var/ww  
10:27:05 [Build Functional CE]   w/html/lib/internal/Magento/Framework/Setup/Patch/PatchRegistry.php on line  
10:27:05 [Build Functional CE]    141                                                                         
10:27:05 [Build Functional CE]                                                                                
10:27:05 [Build Functional CE] 
10:27:05 [Build Functional CE] Exception trace:
10:27:05 [Build Functional CE]   at /var/www/html/lib/internal/Magento/Framework/App/ErrorHandler.php:61
10:27:05 [Build Functional CE]  Magento\Framework\App\ErrorHandler->handler() at n/a:n/a
10:27:05 [Build Functional CE]  call_user_func() at /var/www/html/lib/internal/Magento/Framework/Setup/Patch/PatchRegistry.php:141
10:27:05 [Build Functional CE]  Magento\Framework\Setup\Patch\PatchRegistry->getDependencies() at /var/www/html/lib/internal/Magento/Framework/Setup/Patch/PatchRegistry.php:206
10:27:05 [Build Functional CE]  Magento\Framework\Setup\Patch\PatchRegistry->getIterator() at /var/www/html/lib/internal/Magento/Framework/Setup/Patch/PatchApplier.php:138
10:27:05 [Build Functional CE]  Magento\Framework\Setup\Patch\PatchApplier->applyDataPatch() at /var/www/html/setup/src/Magento/Setup/Model/Installer.php:1094
10:27:05 [Build Functional CE]  Magento\Setup\Model\Installer->handleDBSchemaData() at /var/www/html/setup/src/Magento/Setup/Model/Installer.php:960
10:27:05 [Build Functional CE]  Magento\Setup\Model\Installer->installDataFixtures() at n/a:n/a
10:27:05 [Build Functional CE]  call_user_func_array() at /var/www/html/setup/src/Magento/Setup/Model/Installer.php:389
10:27:05 [Build Functional CE]  Magento\Setup\Model\Installer->install() at /var/www/html/setup/src/Magento/Setup/Console/Command/InstallCommand.php:231
10:27:05 [Build Functional CE]  Magento\Setup\Console\Command\InstallCommand->execute() at /var/www/html/vendor/symfony/console/Command/Command.php:255
10:27:05 [Build Functional CE]  Symfony\Component\Console\Command\Command->run() at /var/www/html/vendor/symfony/console/Application.php:1009
10:27:05 [Build Functional CE]  Symfony\Component\Console\Application->doRunCommand() at /var/www/html/vendor/symfony/console/Application.php:273
10:27:05 [Build Functional CE]  Symfony\Component\Console\Application->doRun() at /var/www/html/lib/internal/Magento/Framework/Console/Cli.php:115
10:27:05 [Build Functional CE]  Magento\Framework\Console\Cli->doRun() at /var/www/html/vendor/symfony/console/Application.php:149
10:27:05 [Build Functional CE]  Symfony\Component\Console\Application->run() at /var/www/html/bin/magento:23
10:27:05 [Build Functional CE] 
```

**Comment by nikolalardev** [comment]
@magento run all tests

**Comment by gabrieldagama** [comment]
@magento run Functional Tests EE,Functional Tests B2B

**Comment by gabrieldagama** [comment]
@magento run all tests

**Comment by gabrieldagama** [comment]
@magento run Unit Tests

**Comment by gabrieldagama** [comment]
@magento run all tests

**Comment by gabrieldagama** [comment]
@magento run Functional Tests B2B,Functional Tests CE,Sample Data Tests CE

**Review by gabrieldagama — APPROVED** [review]
@nikolalardev Thanks for contributing, the changes look good to me.

The failed tests aren't related to the PR changes.

**Comment by magento-engcom-team** [comment]
Hi @gabrieldagama, thank you for the review.
[ENGCOM-8505](https://jira.corp.magento.com/browse/ENGCOM-8505) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**After:** :heavy_check_mark: 

**_Albania_**

after1

**_Denmark_**

after2

**_Greece_**

after3

**_Iceland_**

after4

**_Portugal_**

after5

**_Sweden_**

after6

**Comment by nikolalardev** [comment]
@magento run Functional Tests B2B

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Comment by nikolalardev** [comment]
@magento run all tests

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Comment by nikolalardev** [comment]
@magento run all tests

**Pull request merged**